main.go

```diff
@@ -122,6 +128,18 @@ func parseColumns(cols string) ([]int, error) {
 	return i, nil
 }
 
+// parseHideFs parses the supplied hide-fs flag into a map of fs types which should be skipped.
+func parseHideFs(hideFs string) map[string]struct{} {
+	hideMap := make(map[string]struct{})
+	for _, fs := range strings.Split(hideFs, ",") {
+		if len(fs) == 0 {
+			continue
+		}
+		hideMap[fs] = struct{}{}
```

Comment:
Might be worth ensureing the inputted filetypes are converted to lowercase: e.g.
```suggestion
		hideMap[strings.ToLower(fs)] = struct{}{}
```

Reply:
Not pretty sure whether this value is case-sensitive. 

https://man7.org/linux/man-pages//man5/fs.5.html
> ```
> FILESYSTEMS(5)             Linux Programmer's Manual            FILESYSTEMS(5)
>
> NAME
>        filesystems  -  Linux  filesystem  types:  ext, ext2, ext3, ext4, hpfs,
>        iso9660, JFS, minix, msdos, ncpfs nfs, ntfs, proc, Reiserfs, smb, sysv,
>        umsdos, vfat, XFS, xiafs,
> ```

It seems like fs types are case-sensitive according to the description. 
But I'm confused now, since a XFS filesystem shows `xfs` in `Fstype`, but on that man page it says `XFS`.

Reply:
Did you come to a conclusion here? Personally I think I agree, purely because users shouldn't be required to type `XFS` or `JFS` (all uppercase) on the command-line. This would probably cause a bunch of confusion, especially if the filtering doesn't apply silently: there wouldn't be an error or any indication that lowercase vs uppercase makes a difference here.

Reply:
After an experiment, I found out that the value of `Fstype` **is case-sensitive**. I've managed to create a mount point, with upper-case letter in its type name, by using `hello` example in libfuse.  And this is how I tested:
1. build hello example
2. rename the output binary to `Hello`
3. execute `./Hello /mnt`, or any other mount point

Then `duf --all` shows:
```
...
│ MOUNTED ON               │ SIZE │ USED │ AVAIL │ USE% │ TYPE                   │ FILESYSTEM        │
├──────────────────────────┼──────┼──────┼───────┼──────┼────────────────────────┼───────────────────┤
│ /mnt                     │   0B │   0B │    0B │      │ fuse.Hello             │ Hello             │
...
```

I think it should not be a problem to make `--hide-fs` a case-sensitive flag. In a general scenario, user would specify `--hide-fs` after finding out there are too much unnecessary mount points listed, which means the actual type name is already printed on screen clearly.